src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -28,20 +27,17 @@
 class DelegatingLoader extends BaseDelegatingLoader
 {
     protected $parser;
-    protected $logger;
     private $loading = false;
 
     /**
      * Constructor.
      *
      * @param ControllerNameParser    $parser   A ControllerNameParser instance
-     * @param LoggerInterface         $logger   A LoggerInterface instance
      * @param LoaderResolverInterface $resolver A LoaderResolverInterface instance
      */
-    public function __construct(ControllerNameParser $parser, LoggerInterface $logger = null, LoaderResolverInterface $resolver)
+    public function __construct(ControllerNameParser $parser, LoaderResolverInterface $resolver)
```

Comment:
removing the argument in the middle of the signature is a BC break.

It is required to accept the new API in 2.8 already to ease the migration (and to throw a deprecation warning when using the old signature)